### PR TITLE
add bandit

### PR DESCRIPTION
**Pull request by bdwyer2**

### CODE REVIEW

File: api/predict.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 # Creating a JSON response model: https://flask-restplus.readthedocs.io/en/stable/marshalling.html#the-api-model-factory
 label_prediction = MAX_API.model('LabelPrediction',
-                                 {l: fields.Float(required=True, description='Class probability') for l in class_labels})
+                                 {l: fields.Float(required=True, description='Class probability') for l in class_labels})  # noqa
```
Comment (xuhdev) [inline api/predict.py]: What is the issue of this line?

Comment (xuhdev) [reply api/predict.py]: I got it, it's fine.

```suggestion
                                 {l: fields.Float(required=True, description='Class probability') for l in class_labels})  # noqa: E741
```

Comment (bdwyer2) [reply api/predict.py]: The newest version of `flake8` returns this error, older versions do not:

```
./api/predict.py:33:103: E741 ambiguous variable name 'l'
```

File: api/predict.py [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
 # Creating a JSON response model: https://flask-restplus.readthedocs.io/en/stable/marshalling.html#the-api-model-factory
 label_prediction = MAX_API.model('LabelPrediction',
-                                 {l: fields.Float(required=True, description='Class probability') for l in class_labels})
+                                 {l: fields.Float(required=True, description='Class probability') for l in class_labels})  # noqa: E741
```
Comment (bdwyer2) [inline api/predict.py]:
```suggestion
                                 {l: fields.Float(required=True, description='Class probability') for l in class_labels})  # noqa - E741
```

**Review by xuhdev — APPROVED** [review]

**Pull request merged**